apache-phpbrew/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM php:5.6-apache
```

Comment:
Can you still use phpbrew if you base the image on, say [alpine](https://registry.hub.docker.com/_/alpine/)? That would cut the fat quite a lot.